docs/source/cluster_management/backup-restore.rst

```diff
@@ -73,12 +73,12 @@ These two timestamp files and the entirety of your underlying storage's backup a
 5. Release the Backup Lock
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Call the ``persistent-lock/release`` endpoint, passing back the ``LockEntry`` that you received in step 1.
-For example (replace the serialised ``LockEntry`` with your own from step 1):
+Call the ``persistent-lock/release-backup-lock`` endpoint, passing back the ``PersistentLockId`` that you received in step 1.
+For example (replace the serialised ``PersistentLockId`` with your own from step 1):
```

Comment:
Please change the `Obtain the backup Lock`` to say it returns the ``PersistentLockId``. That should have been a part of #1665.